frontend/src/lib/components/widgets/Multiselect/Multiselect.tsx

```diff
@@ -37,6 +36,7 @@ import { Placement } from "src/lib/components/shared/Tooltip"
 import { VirtualDropdown } from "src/lib/components/shared/Dropdown"
 import { fuzzyFilterSelectOptions } from "src/lib/components/shared/Dropdown/Selectbox"
 import { labelVisibilityProtoValueToEnum } from "src/lib/util/utils"
+import { withTheme } from "@emotion/react"
```

Comment:
Yes, I wanted it to be next to `EmotionTheme`. Do we need to introduce some import ordering tool?

Reply:
Hmm, I see. Yep, I think an import ordering tool would be great. The thing is that it is more consistent with the current codebase if it is on top since it's mostly sorted based on external imports first, and then internal, and last are the relative imports.